config.yaml

```diff
@@ -167,6 +167,14 @@ commands:
 
   N98\Magento\Command\Installer\InstallCommand:
     magento-packages:
+      - name: magento-ce-2.1.2
+        package: magento/project-community-edition
+        version: 2.1.2
+        options:
+          prefer-dist: true
+          stability: alpha
+          repository-url: https://repo.magento.com
+    magento-packages:
```

Comment:
you duplicated "magento-packages:" key here, this makes other packages disappear and the build red. please fix (remotve the duplicate on line 177).